### PR TITLE
Restructure planning documentation

### PR DESCRIPTION
**Pull request by nicksnyder**
This PR does two things
1. It separates the philosophy documentation from the reference documentation for planning.
2. It refreshes some of the content so it is consistent with what has already been communicated in slack messages.

The most important part of this diff is planning-process.md.

### CODE REVIEW

File: content/departments/product-engineering/process/planning-communication.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+[Current vision and strategy.](../strategy-goals/index.md)
+
+## Quarterly department OKRs
+
+Each quarter we set OKRs at the department level (all of product and engineering).
+
+OKRs define what success looks like each quarter, taking into account where we are at as a company and what we think the next things we need to accomplish given the vision and strategy we have for the year.
+
+We don't do OKRs at lower levels (orgs, teams); instead we ask orgs and teams to focus on building out their roadmap (using strategy and OKRs as an input).
```
Comment (jasonyavorsky) [inline content/departments/product-engineering/process/planning-communication.md]: Maybe worth mentioning that this doesn't mean teams are forbidden from setting their own quarterly goals, just that they aren't tracked at the product-engineering level. Feels sort of obvious, but maybe good to be explicit.

File: content/departments/product-engineering/process/planning-communication.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+# Product/Engineering planning communication
+
+This page documents how we communicate our plans and progress.
```
Comment (jasonyavorsky) [inline content/departments/product-engineering/process/planning-communication.md]: Narrative strategy pages are missing from this section, but should be here IMHO as they are important to communicate what we're prioritizing and why, and ties things together on a per team basis beyond the list of items in the roadmap. https://handbook.sourcegraph.com/strategy-goals/strategy#team-strategy-pages. Do you agree?

File: content/departments/product-engineering/process/planning-communication.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+- The email subject should contain the team name and the date (for example: "Search update 2020-10-14") so each update starts a distinct email thread (otherwise, they get grouped in the Google Groups UI).
+- There is no strict format to follow for the content, but here are some tips:
+  - Favor prose over a list of changes.
+  - Be brief and link to details.
+  - Consider sharing wins, challenges, risks, plans, lessons learned.
+  - Inline relevant screenshots/demos/gifs/charts when possible so it isn't just a wall of text.
+  - Be creative and have fun with it! Jokes and random fun facts are welcome.
+
+## Monthly PMM roadmap update
```
Comment (jasonyavorsky) [inline content/departments/product-engineering/process/planning-communication.md]: This (and maybe some other content on this page?) duplicates some of the content in the product management handbook: https://handbook.sourcegraph.com/departments/product-engineering/product/process. In particular https://handbook.sourcegraph.com/departments/product-engineering/product/process#keeping-strategy-up-to-date.

File: content/departments/product-engineering/process/planning-process.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+We expect to not need a lot of formal process here, because if we are effectively using this roadmap as a source of truth, it will organically get updated on a regular basis as part of normal team operations.
+
+## Quarterly department retrospective
+
+Near the end of each quarter, the product/engineering VPs and Directors will get together for a retrospective. The purpose of this retrospective is to create a shared safe space for celebrating wins, learning, and improvement.
+
+- Before the retrospective we will
+  - Solicit feedback from the broader product/engineering team
+  - Asyncronously write down thoughts on topics we may want to discuss during the retrospective. Prompts may include but aren't limited to:
```
Comment (jasonyavorsky) [inline content/departments/product-engineering/process/planning-process.md]:
```suggestion
  - Asynchronously write down thoughts on topics we may want to discuss during the retrospective. Prompts may include but aren't limited to:
```

File: content/departments/product-engineering/process/index.md [hunk 5]
```diff
@@ -1,7 +1,8 @@
 # Product and Engineering Processes
 
-- [Planning](planning.md)
+- [Planning philosophy](planning-philosophy.md)
+- [Planning communication](planning-communication.md)
+- [Planning process](planning-process.md)
```
Comment (jasonyavorsky) [inline content/departments/product-engineering/process/index.md:4]: Planning process and communication might benefit from being on one page given how closely they connect to each other.

**Comment by nicksnyder** [comment]
[Responding to feedback on slack](https://sourcegraph.slack.com/archives/C02KZ941RNX/p1644370811455439?thread_ts=1644369769.763579&cid=C02KZ941RNX)

> The “content/departments/product-engineering/process/planning-communication.md” Looks directionally correct. Thanks Nick.

Great, this was the most important part to me.

> I personally find the “content/departments/product-engineering/process/planning-philosophy.md” to be wordy and much more a reflection of our aspirations rather than a reflection of who we are.  I wonder of all these section headings what is the most important philosophy that we should focus on as we continue to improve our planning process? I also realize the irony of this feedback given that the page name is “philosophy”

I agree and I was pondering deleting the philosophy section myself.

> As for the “content/departments/product-engineering/process/planning-process.md” I think that framing our planning around boundaries like Annual / Quarterly boxes us out of iterative planning.  Also what we have in those sections are just goals to get better at planning.  I like the Roadmap & Restrospective headings, but don’t have great feedback on how we should improve the details in those sections.  I will sleep on it and see if I come up with anything.

Good point around this framing being limiting. The only reason I added this was to make it clear that some of this stuff was explicitly not figured out.

***

Will think about this more today and make some updates to the PR.

File: content/departments/product-engineering/process/index.md (hunk 5 above)
Comment (nicksnyder) [reply content/departments/product-engineering/process/index.md:4]: Agree, will change this

File: content/departments/product-engineering/process/planning-communication.md (hunk 2 above)
Comment (nicksnyder) [reply content/departments/product-engineering/process/planning-communication.md]: Agree, I moved this here.

File: content/departments/product-engineering/process/planning-communication.md (hunk 3 above)
Comment (nicksnyder) [reply content/departments/product-engineering/process/planning-communication.md]: Agree, I moved content here and deleted the other content

File: content/departments/product-engineering/process/planning-communication.md (hunk 1 above)
Comment (nicksnyder) [reply content/departments/product-engineering/process/planning-communication.md]: Does this wording change address your comment?
https://github.com/sourcegraph/handbook/pull/2247/commits/5aad28aa96565441d78663aab2cdd5f77bc85006

**Comment by nicksnyder** [comment]
I made some updates to try to address all the feedback.

For planning philosophy, it is not on a separate page so it doesn't distract when folks want to look up what we actually do.

I am totally open to editing/trimming/deleting philosophy content (and would be happy for someone else to own this and propose edits), but I am treating that as out of scope for this change.

File: content/departments/product-engineering/process/planning-communication.md (hunk 1 above)
Comment (jasonyavorsky) [reply content/departments/product-engineering/process/planning-communication.md]: Yes, thanks!

File: content/departments/product-engineering/process/planning-process.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+### OKR tracking
+
+The source of truth for current OKRs and status of those OKRs is in our [private issue tracker](https://github.com/sourcegraph/product-engineering-tracker/labels/summary).
+
+- This is private because KRs frequently contain sensitive business metrics and customer names).
+- The sanitized public version of our OKRs are [here in the handbook](../../../strategy-goals/goals/index.md).
+
+Each KR has an associated GitHub issue following [this template](https://github.com/sourcegraph/product-engineering-tracker/issues/new?template=product-engineering-kr.md).
+
+- The assignee of each OKR is responsible for posting an update to the associated tracking issue by 19:00 UTC each Friday. Weekly updates should include:
```
Comment (thiswayman) [inline content/departments/product-engineering/process/planning-process.md]: I think this is the area I had a bit of confusion on...

_The assignee of each OKR is responsible for posting an update to the associated tracking issue by 19:00 UTC each Friday._ To post an update, create a new comment and include the current status along with the following details:

Comment (nicksnyder) [reply content/departments/product-engineering/process/planning-process.md]:
```suggestion
- The assignee of each OKR is responsible for posting an update to the associated tracking issue by 19:00 UTC each Friday. To post an update, create a new comment with the following details:
```

**Pull request merged**